lib/cmds/files_cmds/mv.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+const chalk = require('chalk');
+const querystring = require('querystring');
+const path = require('path');
+const { patch, list } = require('../../api');
+
+exports.command = 'mv <source> <dest>';
+exports.desc = 'Move/rename files from <source> to <dest> within a project';
+exports.builder = yargs => {
+  yargs.positional('source', {
+    describe: 'Either the ID of the file to move or a project ID and prefix of the form <project ID>/<prefix> to move a set of files in a project.',
+    type: 'string'
+  }).positional('dest', {
+    describe: 'The destination directory for the download file.  Defaults to current working directory',
+    type: 'string'
+  }).option('recursive', {
+    describe:
+      'If <source> is a project Id with an optional prefix, move all files under that prefix to the destination.',
+    alias: 'r',
+    type: 'boolean',
+    default: false
+  }).option('limit', {
+    describe: 'The maximum number of files to move when used with the --recursive option',
+    alias: 'l',
+    type: 'number',
+    default: 1000
+  });
+};
+
+exports.handler = async argv => {
+  if (argv.recursive) {
+    const [datasetId, prefix] = argv.source.split('/');
+
+    const opts = {
+      datasetId,
+      pageSize: argv.limit
+    };
+
+    if (prefix) {
+      opts.name = prefix;
+    }
+
+    const response = await list(argv, `/v1/files?${querystring.stringify(opts)}`);
+    for (const item of response.data.items) {
+      const newName = path.join(argv.dest, item.name.replace(prefix, ''));
```

Comment:
On windows `path.join()` is going to use `\` rather than `/` which isn't what we want.

Reply:
Good catch.  Fixed.